core/domain/exp_fetchers_test.py

```diff
@@ -253,43 +252,6 @@ def test_retrieval_of_multiple_explorations(self):
             exp_fetchers.get_multiple_explorations_by_id(
                 exp_ids + ['doesnt_exist'])
 
-    def test_exploration_user_data_is_none_before_starting_exploration(self):
```

Comment:
What happened to these tests?

Reply:
Ah, I checked out this file from an older branch I was working on. Fixed now!